examples/sg-accel-index-reader.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I feel like these should be moved to the sg-accel examples folder.

This one could be renamed 'sync-gateway-accel-enabled.json'.